### ISSUE TITLE
Array errors missing when minLength is set on content?

### ISSUE THREAD

**Issue by ciscoheat** [fabian-hiller/valibot#288]
I've noticed that for a schema like this:

```ts
const arr = array(string([minLength(2)]), [minLength(2)]);
console.log(safeParse(arr, ['A']));
```

There is only one issue reported:

```ts
issues: [                         
  {                               
    reason: 'string',             
    validation: 'min_length',     
    origin: 'value',              
    message: 'Invalid length',    
    input: 'A',                   
    path: [Array],                
    abortEarly: undefined,        
    abortPipeEarly: undefined,    
    skipPipe: undefined           
  }                               
]                                 
```

I'd expected that both the array and the string issue is reported? That's the behavior of Zod and Ajv, at least.

**Comment by fabian-hiller** [comment]
This should be fixed in v0.22.0

**Comment by ciscoheat** [comment]
It is, thank you very much!
